index.js

```diff
@@ -570,4 +574,54 @@ input UpdateMaxValueInput {
   inclusive: Boolean!
   custom: Int
 }
+
+type Metadata {
+    id: ID!
+    key: String
+    alias: String
+    type: MetadataType!
+    dateValue: Date
+    regionValue: Region
+    languageValue: Language
+    textValue: String
+}
+
+enum MetadataType {
+    Date
+    Text
+    Region
+    Language
+}
+
+enum Region {
+    GB_ENG
+    GB_GBN
+    GB_NIR
+    GB_SCT
+    GB_WLS
+}
+
+enum Language {
+    en
+    cy
+}
+
```

Comment:
Should the types and enums be above the mutations to keep with the existing schema format?